tox.ini

```diff
@@ -92,4 +92,13 @@ description = Run integration tests
 deps =
     pytest
 commands =
-    pytest -v --tb native --ignore={[vars]tst_dir}unit --log-cli-level=INFO -s {posargs}
+    pytest -v --tb native --ignore={[vars]tst_dir}unit \
+        --ignore={[vars]tst_dir}integration/test_dnf.py \
+        --log-cli-level=INFO -s {posargs}
+
+[testenv:integration-but-with-cleantest]
+description = Run integration tests using cleantest
+deps =
+    cleantest==0.4.0-rc1
```

Comment:
I am not bringing pytest in as a dependency here because I am using the `unittest` module in this case. I did not want to super befuddle the tests before discussing the tests for the dnf charm library.

Reply:
Opened issue https://github.com/canonical/operator-libs-linux/issues/76 so we can continue the discussion about testing charm libraries here.